### ISSUE TITLE
rubidium version (forge)

### ISSUE THREAD

**Issue by benjaminkitson** [yezhiyi9670/occlusion-culling-fix#1]
I know asking about forge version of fabric mods often doesn't go down well, but given how small of a change this makes I thought I'd ask - how plausible would it be to implement this exact change but for forge? Cheers

**Comment by yezhiyi9670** [comment]
The technical details should be exacly same, but I don't have much free time to do that. Maybe later.

**Comment by benjaminkitson** [comment]
yeah that's the one - okay, no worries, having the issue fixed so seamlessly in sodium and then finding it again in rubidium is infuriating 😂 cheers for the original mod though! I'd love to see it in forge, but yeah i get not having much free time, thanks for the quick response 👍
